sdk/go/gcp/accesscontextmanager/pulumiManifest.go

```diff
@@ -0,0 +1,13 @@
+// *** WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. ***
```

Comment:
I think you should gitignore these files

Reply:
yeah I am just doing that right now :)